source/bson-corpus/tests/double.json

```diff
@@ -28,16 +28,16 @@
             "relaxed_extjson": "{\"d\" : -1.0001220703125}"
         },
         {
-            "description": "1.23456789012345677E+18",
+            "description": "1.2345678901234568e+18 - precision adjusted for Ruby",
             "canonical_bson": "1000000001640081E97DF41022B14300",
-            "canonical_extjson": "{\"d\" : {\"$numberDouble\": \"1.23456789012345677E+18\"}}",
-            "relaxed_extjson": "{\"d\" : 1.23456789012345677E+18}"
+            "canonical_extjson": "{\"d\" : {\"$numberDouble\": \"1.2345678901234568e+18\"}}",
+            "relaxed_extjson": "{\"d\" : 1.23456789012345678e+18}"
```

Comment:
I suspect that this test will now fail in other languages besides Python/Ruby because some actually do output ` 1.23456789012345677E+18`. The best option might be to change `canonical_bson` to a large double that is serialized to JSON the same across all drivers. `1.0001220703125` seems to work the same in all drivers (no one has complained about those tests yet). How about we try `1.0001220703125E+18`?

```python
>>> 1.0001220703125E+18
1.0001220703125e+18
```

Reply:
This value passes tests in Ruby, I updated the PR.